examples/petstore/controllers/pets.go

```diff
@@ -126,6 +131,26 @@ func (rs PetsResources) filterPets(c fuego.ContextNoBody) ([]models.Pets, error)
 	})
 }
 
+type FilterParams struct {
+	Name        string `query:"name" description:"Filter by name" example:"cat name"`
+	YoungerThan int    `query:"younger_than"`
+	JustAnArray []int  `query:"just_an_array" example:"1,2,3"`
+}
+
+func (rs PetsResources) filterPetsStronglyTyped(c fuego.ContextWithParams[FilterParams]) ([]models.Pets, error) {
+	params, err := c.Params()
+	if err != nil {
+		return nil, err
+	}
+
+	fmt.Println("params", params)
```

Comment:
Leave this on accident?

Reply:
No it was for debugging the request. 

What do you want to do with the new slice field?

Reply:
Ah i see